jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -212,6 +212,8 @@ static PluginConfigurationProcessor processCommonConfiguration(
           LogEvent.warn(
               "Setting image creation time to current time; your image may not be reproducible."));
       jibContainerBuilder.setCreationTime(Instant.now());
+    } else {
+      jibContainerBuilder.setCreationTime(getConfiguredTime(rawConfiguration.getCreationTime()));
```

Comment:
For extraDirectories, I think I throw an error if both extraDirectory and extraDirectories are given to eliminate any possible pitfall while the user is migrating. I think it's worth the effort to prevent such an unfortunate peril that may otherwise not be easily detected.

Reply:
Yeah I'm thinking about going back over this logic, although at the same time I feel like the deprecation warning might be enough to clue people into the problem if they're getting unexpected results with both configuration options present.

Reply:
The difference is that the user will get to know something is wrong far later at runtime when a wrong timestamp matters and it may actually take some time until they notice there is a problem. And people may not really pay much attention to check all warnings. It feels bad to make the code complex for a tiny thing, but they will be gone once we get to 2.0, and considering the benefit, I think it's worth. I think it could really preventing people from opening new issues from this.